src/editors/playgroundController.ts

```diff
@@ -93,6 +93,24 @@ export default class PlaygroundController {
 
   runAllPlaygroundBlocks(): Promise<boolean> {
     return new Promise(async (resolve) => {
+      const activeConnection = this._connectionController.getActiveDataService();
+      const shouldConfirmRunAll = vscode.workspace
+        .getConfiguration('mdb')
+        .get('confirmRunAll');
+
+      if (activeConnection && shouldConfirmRunAll === true) {
+        const name = this._connectionController.getActiveConnectionName();
+        const confirmRunAll = await vscode.window.showInformationMessage(
+          `Are you sure you with to run this playground against ${name}?`,
```

Comment:
What are your thoughts on adding a second line here:
`Are you sure you with to run this playground on connection ${name}? This confirmation can be disabled in the extension settings.`